pkg/detectors/openapi/openapi.go

```diff
@@ -44,7 +44,7 @@ func (detector *detector) ProcessFile(file *file.FileInfo, dir *file.Path, repor
 
 	fileType, err := getFileType(file)
 	if err != nil {
-		return false, err
+		return false, nil
```

Comment:
Is this intentional? If so, can we log the error (as debug)?

Reply:
Good idea!